R/plot_taxa.R

```diff
@@ -1236,51 +1236,72 @@ plot_taxa_abund <- function(data,
 
 
 
-#' Plot sites on US map
+#' Work-in-progress upgraded plot_sites
 #'
 #' @param data (list or tbl_df, tbl, data.frame) The dataset object returned by \code{read_data()}, a named list of tables containing the observation and taxon tables, or a flat table containing columns of the observation and location tables.
 #' @param id (character) Identifier of dataset to be used in plot subtitles. Is automatically assigned when \code{data} is a dataset object containing the \code{id} field, or is a table containing the package_id column.
 #' @param alpha (numeric) Alpha-transparency scale of data points. Useful when many data points overlap. Allowed values are between 0 and 1, where 1 is 100\% opaque. Default is 1.
-#' @param labels (logical) Argument to show labels of each US state. Default is TRUE.
-#' 
+#' @param color_var (character) Argument to assign colors to the points on the plot
```

Comment:
@sokole @RafaelRangel0 I'm wondering if names like `point_color` and `point_shape` could be more informative.

Reply:
sure, we were trying to conform to the existing naming conventions for other plotting functions in the package, but others do not have these sorts of arguments.